app/less/components.less

```diff
@@ -92,3 +92,18 @@ pick-a-date, pick-a-time {
     }
   }
 }
+
+/****** min and max date styles *****/
+.date-alignment{
+    padding-top: 6px;
+}
+.valid-range-msg{
+ min-width:97px;
+ float:left;
+ font-weight:normal;
+ padding-left:15px;
```

Comment:
You don't need this if you put the content in a Bootstrap column as mentioned below.